### DISCUSSION TITLE
Reanimated 3.4.0 release notes

### DISCUSSION THREAD

**Discussion by parjanyaacoder** [software-mansion/react-native-reanimated#4819] [Q&A] [answered]
The release notes for [Reanimated v3.4.0](https://github.com/software-mansion/react-native-reanimated/releases/tag/3.4.0) mentions requirements for Shared Element Transitions as 
`Added support for progress based transition (⚠️ requires react-native-screens@3.23.0 +)`

But the latest version of react-native-screens is 3.22.1 as per the release [notes](https://github.com/software-mansion/react-native-screens/releases) of `react-native-screens`. 

Is this a typo in release docs and should I create an issue for this ? Is there any example of progress based transition in Shared Element Transition ?

**Comment by piaskowyk** [comment, marked as answer]
Hey! No, it wasn't a mistake. The `react-native-screens` has just released https://github.com/software-mansion/react-native-screens/releases/tag/3.23.0 😅

When it comes to an examples you can find them here: https://github.com/software-mansion/react-native-reanimated/tree/main/app/src/examples/SharedElementTransitions

**Reply by parjanyaacoder** [reply]
Thanks for your answer. `react-native-screens v3.23.0` is released 1 hour ago. I was trying before this release. 😄 I found it so intriguing that I didn't wait for the release. And thanks for the examples link.
